juriscraper/scotus/scotus_docket_report.py

```diff
@@ -0,0 +1,413 @@
+                  "type":"Petitioner"
+               }
+            ]
+
+        :return: List of parties dicts.
+        """
+        parties = []
+        parties.extend(self._build_parties_for_type("Petitioner"))
+        parties.extend(self._build_parties_for_type("Respondent"))
+        parties.extend(self._build_parties_for_type("Other"))
+        return parties
+
+    @classmethod
+    def normalize_date(
+        cls, date_str: Optional[str]
+    ) -> Optional[datetime.date]:
+        """Convert a date string to YYYY-MM-DD using common SCOTUS formats.
+
+        :param date_str: Raw date string (e.g., "July 3, 2024", "Jul 03 2024", "08/16/2024")
+        :return: A date python object or None
+        """
+        if not date_str:
+            return None
+        s = date_str.strip()
+        for date_format in cls.DATE_FORMATS:
+            try:
+                return datetime.strptime(s, date_format).date()
+            except ValueError:
+                continue
+        return None
+
+    @staticmethod
+    def _apply_prefix(prefix: str, parts: list[str]) -> list[str]:
+        """Prepend prefix to numeric parts."""
+        return [
+            f"{prefix}{p.strip()}" if p.strip().isdigit() else p.strip()
+            for p in parts
+        ]
+
+    @staticmethod
+    def _clean_parts(parts: list[str]) -> list[str]:
+        """Clean lower court numbers
+
+        :param parts: list of strings
+        :return: list of cleaned lower court numbers
+        """
+        return [p.strip() for p in parts]
+
+    def clean_lower_court_cases(self, raw_text: str) -> list[str]:
+        """Clean and normalize lower court case numbers from various formats.
+
+        :param raw_text: Raw text to be cleaned.
+        :return: A list of lower court case numbers.
+        """
+        raw_text = raw_text.replace("(", "").replace(")", "").strip()
+
+        # Case 1: Slash-separated numbers with same prefix
+        if self.LOWER_COURT_PATTERNS["slash_same_prefix"].match(raw_text):
+            parts = raw_text.split("/")
+            prefix_match = re.match(r"(.*-)(\d+)$", parts[0])
+            if not prefix_match:
+                return raw_text.split(",")
+            prefix = prefix_match.group(1)
+            return self._clean_parts(
+                self._apply_prefix(prefix, [p.lstrip("-") for p in parts])
+            )
+
+        # Case 2: Various separators with inherited prefixes: "98-4033;-4214;-4246"
+        if self.LOWER_COURT_PATTERNS["sep_inherited_prefix"].match(raw_text):
+            parts = re.split(r"[;/,-]", raw_text)
+            prefix = parts[0].split("-")[0]
+            return self._clean_parts(
+                [
+                    f"{prefix}-{p.strip('-')}"
+                    for p in parts
+                    if p.strip("-") != prefix
+                ]
+            )
+
+        # Case 3: Negative prefix-separated numbers: "99-1845,-1846,-1847,-197" or "98-60240,-60454,-60467,-""
```

Comment:
I believe there's an extra quotation mark at the end of the comment.

```suggestion
        # Case 3: Negative prefix-separated numbers: "99-1845,-1846,-1847,-197" or "98-60240,-60454,-60467,-"
```